generator-fireloop/src/sdk/index.ts

```diff
@@ -12,89 +14,116 @@ import * as chalk from 'chalk';
  */
 module.exports = generators.extend({
   /**
-   * @method prompting
-   * @author Brannon N. Darby II <gh:ng-logic>
-   * @description
-   * If this.options.showOptions = false, build SDK with default options
-   * Otherwise, prompt user with options and use selections when building SDK
+   * @module fireloop:sdk
+   * @author Brannon N. Darby II <gh:brannon-darby>
    */
-  prompting: function () {
-    let keys: { IO: string, FIRELOOP: string, DEFAULT_VALUES: string } = {
-      IO: 'Enable PubSub + IO + FireLoop functionality',
-      FIRELOOP: 'Generate ONLY FireLoop SDK + Auth Services',
-      DEFAULT_VALUES: 'Add default values in models'
-    };
-    let choices: string[] = [keys.IO, keys.DEFAULT_VALUES];
-    // TODO: set FIRELOOP to true after fix
-    let selected: { IO: boolean, FIRELOOP: boolean, DEFAULT_VALUES: boolean } = {
-      IO: true,
-      FIRELOOP: false, // FireLoop only will avoid to create LoopBack services, by default we want both of them.
-      DEFAULT_VALUES: false
+  constructor: function() {
+    generators.apply(this, arguments);
+    this.log(yosay('Let\'s build an SDK!'));
+  },
+
+  prompting: function() {
+
+    this.options.clients = this.config.get('clients') || {};
+    // Filter clients only not server.
+    let clients: string[] = [];
```

Comment:
This same logic is done in the other file (generator-fireloop/src/app/index.ts). It might make sense to consider moving it out to some helper method.